packages/flutter/lib/src/material/search.dart

```diff
@@ -370,7 +370,9 @@ abstract class SearchDelegate<T> {
   /// Defaults to the default value specified in [TextField].
   final TextInputType? keyboardType;
 
-  /// {@macro flutter.widgets.editableText.autocorrect}
+  /// Whether to enable autocorrection.
+  ///
+  /// Defaults to true.
```

Comment:
SearchDelegate does not have autofillHints, so default is always true.

Reply:
Yeah makes sense that it's less likely that someone wants to autofill a search field